incubator/goldfish/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
W've started namespacing template. See #1785. Please apply this to your PR.

`{{- define "goldfish.name" -}}`
`{{- define "goldfish.fullname" -}}`

Reply:
thanks for the comment, I've fixed the issue mentioned above, please check again @unguiculus .